examples/with-firebase/pages/profile/[username].js

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head'
+import Error from 'next/error'
+
+import { getProfileData } from '../../fetchData/getProfileData'
+
+export default function SSRPage({ data }) {
+  const { username, profileDataJson } = data
+  const profileData = JSON.parse(profileDataJson)
+
+  if (!profileData) {
+    return <Error statusCode={404} />
+  }
```

Comment:
this can be removed once `notFound` is added to `getServerSideProps`

Reply:
Thank you for your comment. Fixed!